content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,214 @@
+
+### Query Guidance
+
+It is important to design your query to return 1 row. A good practice is to add "| limit 1" at the end of your query.
+
+Data types of your query result should be: real, int or long. Other data types are not supported. Later, during runtime, your data will be converted to int64.
+
+Be careful with setting up "pollingInterval" and long running queries. Test your query before.
+
+### Authentication Parameters
+
+You can use the `TriggerAuthentication` CRD to configure the authentication by providing a set of Azure Active Directory credentials or by using pod identity.
+
+**Credential based authentication:**
+
+- `clientId` - Application id from your Azure AD Application/service principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your service principal.
+- `clientSecret` - Password from your Azure AD Application/service principal.
+- `tenantId` - Azure Active Directory tenant id. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to retrieve your tenant id.
+
+The principal will need `DatabaseViewer` role to  query metrics from the Data Explorer Cluster. Follow [this](https://docs.microsoft.com/en-us/cli/azure/kusto/database?view=azure-cli-latest#az-kusto-database-add-principal) to assign your principal with the right access permissions through az cli.
+
+**Pod identity based authentication:**
+
+[Azure Active Directory pod-managed identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) can be used.
+The identity that being used by pod identity will need `DatabaseViewer` role to particular database query from the Data Explorer Cluster. Follow [this](https://docs.microsoft.com/en-us/cli/azure/kusto/database?view=azure-cli-latest#az-kusto-database-add-principal) to assign your principal with the right access permissions through az cli.
+
+in place of credential based authentication. The following section contains an example of a `TriggerAuthentication` using pod identity.
+
+### Example
+
+The following example illustrates the use of a TriggerAuthentication to connect to Data Explorer.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-data-explorer-secret
+data:
+  clientId: <clientId> # Base64 encoded 
+  clientSecret: <clientSecret> # Base64 encoded 
+  tenantId: <tenantId> # Base64 encoded 
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: azure-data-explorer-trigger-auth
+spec:
+  secretTargetRef:
+    - parameter: clientId
+      name: azure-data-explorer-secret
+      key: clientId
+    - parameter: clientSecret
+      name: azure-data-explorer-secret
+      key: clientSecret
+    - parameter: tenantId
+      name: azure-data-explorer-secret
+      key: tenantId
+  # or Pod Identity, kind: Secret is not required in case of pod Identity
+  podIdentity:
+      provider: azure
```

Comment:
I would choose one or the other, otherwise people might be confused.

Feel free to create two samples similar to https://keda.sh/docs/2.6/scalers/azure-log-analytics/#example if you want